cloudinit/config/schemas/schema-cloud-config-v1.json

```
@@ -421,6 +421,48 @@
         ]
       }
     },
+    "merge_defintion": {
+      "oneOf": [
+        {
+          "type": "string"
+        },
+        {
+          "type": "array",
+          "minItems": 1,
+          "items": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
```

we need a required definition on these objects
```suggestion
            "required": ["nane", "settings"],
            "properties": {
```